LogMonitor/src/LogMonitor/ProcessMonitor.cpp

```diff
@@ -198,6 +289,9 @@ DWORD ReadFromPipe(LPVOID Param)
         {
             break;
         }
+        // add null terminator at the end, dwRead will always be < BUFSIZE
+        chBuf[dwRead] = '\0';
+        clearBuffer(chBuf);

```

Comment:
nit: this should come before loop `break`; I've updated in the `v2/json-output-all` branch.